src/pages/NoticeRegisterPage.tsx

```diff
@@ -1,3 +1,190 @@
+import { useState } from "react";
+
+import DatePicker from "react-datepicker";
+import { useNavigate } from "react-router-dom";
+
+import "react-datepicker/dist/react-datepicker.css";
+
+import { postNotice } from "@/apis/services/noticeService";
+import { Close } from "@/assets/icon";
+import Button from "@/components/Button";
+import TextField from "@/components/TextField";
+import { useUserStore } from "@/hooks/useUserStore";
+import { extractDigits, numberCommaFormatter } from "@/utils/number";
+
+type FormType = {
+  hourlyPay: string;
+  startsAt: Date | null;
+  workhour: string;
+  description: string;
+};
+
+const FIELD_LABELS: Record<keyof FormType, string> = {
+  hourlyPay: "시급",
+  startsAt: "시작 일시",
+  workhour: "업무 시간",
+  description: "공고 설명",
+};
+
 export default function NoticeRegisterPage() {
-  return <div>NoticeRegisterPage</div>;
+  const navigate = useNavigate();
+  const { user } = useUserStore();
+  const [isSubmitting, setIsSubmitting] = useState(false);
+
+  const [form, setForm] = useState<FormType>({
+    hourlyPay: "",
+    startsAt: null,
+    workhour: "",
+    description: "",
+  });
+
+  const handleChange = (
+    key: keyof typeof form,
```

Comment:
```suggestion
    key: keyof FormType,
```
이것도 FormType으로 변경하면 좋을 것 같아요 😅
통일성...!? 때문에요..! 하핳;